src/applications/vaos/routes.jsx

```diff
@@ -29,6 +29,9 @@ export default function createRoutesWithStore(store) {
   return (
     <ErrorBoundary fullWidth>
       <VAOSApp>
+        <div hidden>
+          <span id="new-tab-msg-1">Link opens in a new tab</span>
+        </div>
```

Comment:
Not sure if this is the best place for this but this holds all of the static messages (only 1 right now) that can be referenced when using aria-describedBy.